src/backend/core/api/viewsets.py

```diff
@@ -2,6 +2,7 @@
 
 from django.contrib.postgres.search import TrigramSimilarity
 from django.db.models import Func, Max, OuterRef, Prefetch, Q, Subquery, Value
+from django.db.models.functions import Coalesce
```

Comment:
in commit message: 
s/Triagram/Trigram

Reply:
I didn't understand this statement in the commit message:
> Thus, the total score would reflect
the non-none similarity score.

Reply:
thx for your feedback ! I have updated commit's body